email/emailer.go

```diff
@@ -12,16 +10,13 @@ import (
 // CharSet is a default AWS character set
 const CharSet = "UTF-8"
 
-var awsSession *session.Session
-
-// Init creates a new session for SES requests
-func Init() error {
-	var err error
-	awsSession, err = session.NewSession(&aws.Config{
-		Region:      aws.String("us-west-2"),
-		Credentials: credentials.NewEnvCredentials(),
-	})
-	return err
+// NewSession initialises an AWS session
+func NewSession(sesRegion string) (*session.Session, error) {
```

Comment:
Is this wrapper method really necessary?

Reply:
No, but it provides a default method of starting a session if the client wants to plug & play the package

Reply:
All that method is really doing now is hiding the call to `credentials.NewEnvCredentials()`. If you want to abstract away the AWS stuff, you should wrap the AWS session struct entirely.

Reply:
Something like this (`mailer` contains the AWS session)

```
mailer, err := email.NewClient(cfg...)
iferr...

mailer.Send(email.Email{
  To:      "",
  Subject: "",
  ...
})
```

Reply:
If you did this you could also make a "MustSend" version or equivalent that does logging and no error return too